lib/vsc/utils/timestamp.py

```diff
@@ -105,7 +106,11 @@ def read_timestamp(filename):
 
     """
     cache = FileCache(filename)
-    (_, timestamp) = cache.load('timestamp')
+    try:
+        (_, timestamp) = cache.load('timestamp')
+    except TypeError:
+        logging.warning('could not load timestamp from cache file %s', filename)
+        timestamp = None
```

Comment:
And the effect for scripts using this?

Reply:
also, why would a filecache ever be empty? this looks like a serious error, not some harmless warning

Reply:
then add test for existence, to distinguish between corrupt and never initialised

Reply:
If there's nothing in it, it will be empty as well. Not sure how to check for corrupt.